### PR TITLE
Dance Machine now obeys instrument preference

### PR DESCRIPTION
**Pull request by optimumtact**
:cl: oranges
tweak: Dance machine will not play tracks to you if you have instruments turned off
/:cl:

### CODE REVIEW

File: code/game/machinery/dance_machine.dm [hunk 1]
```diff
@@ -459,6 +459,8 @@
 		var/sound/song_played = sound(selection.song_path)
 
 		for(var/mob/M in range(10,src))
+			if(!M.client || (!M.client.prefs.toggles & SOUND_INSTRUMENTS))
```
Comment (lzimann) [inline code/game/machinery/dance_machine.dm]: the ! needs to be outside parenthesis on the flag check

**Review by Cyberboss — APPROVED** [review]

**Comment by coiax** [comment]
Speed merged...

**Comment by optimumtact** [comment]
nothing to do with me, not sure why they did that

**Comment by ShizCalev** [comment]
Had this open with a bunch of other tabs to review, was only off by a little though so w/e

**Pull request merged**